pyiron_base/job/generic.py

```diff
@@ -524,30 +526,27 @@ def copy_to(self, project=None, new_job_name=None, input_only=False, new_databas
         Returns:
             GenericJob: GenericJob object pointing to the new location.
         """
-        # Update flags
-        if input_only and new_database_entry:
-            new_database_entry = False
-
         # Call the copy_to() function defined in the JobCore
-        new_job_core, file_project, hdf5_project, reload_flag = self._internal_copy_to(
+        new_job_core, file_project, hdf5_project, reloaded = self._internal_copy_to(
             project=project,
             new_job_name=new_job_name,
             new_database_entry=new_database_entry,
             copy_files=False,
             delete_existing_job=delete_existing_job
         )
-        if reload_flag:
-            return new_job_core
-
-        # Remove output if it should not be copied
-        if input_only:
-            if "output" in new_job_core.project_hdf5.list_groups():
-                del new_job_core.project_hdf5[
-                    posixpath.join(new_job_core.project_hdf5.h5_path, "output")
-                ]
-            new_job_core.status.initialized = True
+        new_job_core._copy_to(new_database_entry=new_database_entry, reloaded=reloaded)
         return new_job_core
 
+    def _copy_to(self, new_database_entry, reloaded):
```

Comment:
Can you rename this function to be a bit more specific? Class specific copy to or anything similar would be fine.

Reply:
It's called `_specific_copy_to` now.

Reply:
Honestly I think I liked `_after_copy_to` most so far since it was even more descriptive. What about combining these two ideas for maximum descriptivity: `_after_generic_copy_to`?